test/Service/AuthorizationServiceTest.php

```diff
@@ -157,15 +160,15 @@ public function testReturnsTrueForIdentityWhenHasPermissionButNoAssertionsExists
 
     public function testUsesAssertionsAsInstances()
     {
-        $role = new FlatRole('admin');
-        $identity = new Identity([$role]);
+        $role      = new FlatRole('admin');
```

Comment:
another problem with this is that for example my IDE is configured to  not do this automatically so I have to align by hand.. or will have to change config for this project.. but I guess @prolic can add vote for this :D vote for everything :D because that's maybe only me feels this way